### DISCUSSION TITLE
Not a BUG - Question: form validate method

### DISCUSSION THREAD

**Discussion by doraemonxxx** [vbenjs/vue-vben-admin#3114] [General]
Question:

How does the form validation method work? When I read the function In the codebase, it does not tell how to return the object of the field and value.

Thank you!

**Comment by wangjue666** [comment]
Just look at https://github.com/vbenjs/vue-vben-admin/blob/e25af8dd8c85f1d33ba2e7a1c60c83d5e488a036/src/components/Form/src/hooks/useFormEvents.ts#L347

**Reply by doraemonxxx** [reply]
thank you
